guide/05-working-with-the-spatially-enabled-dataframe/part3_data_io_writing_data.ipynb

```diff
@@ -0,0 +1,1816 @@
+{
```

Comment:
<p>Data in a Spatially enabled DataFrame can be exported to local spatial file formats, such as <a href="http://desktop.arcgis.com/en/arcmap/latest/manage-data/feature-classes/a-quick-tour-of-feature-classes.htm" target="_blank">Feature classes</a> or <a href="http://desktop.arcgis.com/en/arcmap/latest/manage-data/shapefiles/what-is-a-shapefile.htm" target="_blank">shapefiles</a>, and non-spatial formats, such as <a href="https://en.wikipedia.org/wiki/Comma-separated_values" target="_blank">csv</a> files or <a href="https://pro.arcgis.com/en/pro-app/latest/help/data/geodatabases/overview/table-basics.htm" target="_blank">tables</a>. Let's take a look.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>